node-perf-dash/downloader.go

```diff
@@ -69,10 +71,43 @@ func (d *LocalDownloader) GetLastestBuildNumber(job string) (int, error) {
 
 // ListFilesInBuild returns the contents of the files with the specified prefix
 // for the test job at the given buildNumber.
-//
-// TODO(yguo0905): Implement this function.
 func (d *LocalDownloader) ListFilesInBuild(job string, buildNumber int, prefix string) ([]string, error) {
-	return nil, fmt.Errorf("ListFilesInBuild() is not yet implemented for local downloader")
+	filesInDir, err := filesInLocalDir(path.Join(*localDataDir, fmt.Sprintf("%d", buildNumber)), "", 0, 10)
+	if err != nil {
+		return nil, err
+	}
+	filesInBuild := []string{}
+	for _, file := range filesInDir {
+		if strings.HasPrefix(file, prefix) {
+			filesInBuild = append(filesInBuild, file)
+		}
+	}
+	return filesInBuild, nil
+}
+
+func filesInLocalDir(baseDir, extraDirs string, currentDepth, maxDepth int) ([]string, error) {
```

Comment:
The `ListFilesInBuild` function is called with `prefix` set to `artifacts/performance-*`
(https://github.com/kubernetes/contrib/blob/master/node-perf-dash/parser.go#L91-L92), so we only need to search the files under `$MY_TEST_RESULT_PATH/$BUILD_NUMBER/artifacts`, rather than recursively?